### PR TITLE
chore: remove CssTemplate and Annotation access from gamma role

### PR DESCRIPTION
**Pull request by lilykuang**
### SUMMARY

- gamma user should not have access to csstemplatemodelview/list/ and annotationlayer/list/ .

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by john-bodley** [comment]
@lilykuang what's the difference (from a Gamma role perspective) between providing access to these lists and say charts or dashboards.

**Comment by lilykuang** [comment]
@john-bodley this pr is intend to fix following Apache Superset vulnerabilities 
https://trello.com/c/CtZKRFHQ/57-fwd-apache-superset-issues-and-vulnerabilities-2-annotationlayer-list

**Comment by eschutho** [comment]
> @lilykuang what's the difference (from a Gamma role perspective) between providing access to these lists and say charts or dashboards.

@john-bodley this is one of those odd cases where the user doesn't have list/menu view access, but has access to the underlying data, which they shouldn't have. So it's just aligning the existing view perms with the underlying data.

File: superset/security/manager.py [hunk 1]
```diff
@@ -179,19 +177,21 @@ class SupersetSecurityManager(  # pylint: disable=too-many-public-methods
...
+        "CssTemplate",
+        "CsvToDatabaseView",
         "ExcelToDatabaseView",
+        "Import dashboards",
         "ImportExportRestApi",
+        "Manage",
+        "Queries",
+        "ReportSchedule",
+        "TableSchemaView",
+        "Upload a CSV",
```
Comment (eschutho) [inline superset/security/manager.py:194]: yey for alphabetization!! 🙏

**Review by eschutho — APPROVED** [review]
LGTM!

**Comment by villebro** [comment]
Sorry for being late to the party @lilykuang @eschutho , but would it be a good idea to include this change in `UPDATING.md`, as it may break current workflows for some existing users/deployments?

**Comment by john-bodley** [comment]
@michael-s-molina if, per @villebro's comment, this is deemed breaking change, wouldn't this need to wait until Superset 4.0?

**Comment by villebro** [comment]
@lilykuang @eschutho @john-bodley @michael-s-molina this may well be murky, and I'm not personally super opinionated one way or another, as I don't think these are commonly used perms. but in the past I think we've erred on the side of caution to be as semver compliant as possible.

**Comment by eschutho** [comment]
I agree that if we think it's a breaking change, then we should wait until 4.0, but my understanding was that the user cannot access these menus from the client, so I don't believe that this change should impact any current workflows. Not sure @lilykuang if you have any other context. Where it does break functionality, is for api usage on these endpoints for /csstemplatemodelview/ and /annotationlayer/ which similar to what @villebro mentioned about the perms, these endpoints aren't widely used. We could leave it in master, but take it out of 2.1.2 or take it out of master and put it in 4.0, or just leave as is. @dpgaspar may have an opinion on this as well.

**Comment by dpgaspar** [comment]
This is actually a bug. A Gamma user should not have access to these views and users don't have access to it via the UI. Really don't view it as a breaking change, but a note on UPDATING would have been nice. This was included on 3.0

**Comment by michael-s-molina** [comment]
I agree with @dpgaspar here. I don't view this as a breaking change as we shouldn't have granted the access in the first place. It will break current workflows as @villebro mentioned but that's exactly what I expect this change to do given that these users are accessing resources they shouldn't have access to.

**Comment by villebro** [comment]
Thanks for the comments @dpgaspar and @michael-s-molina - in other words it sounds like this can be thought of as a security fix of sorts. For future reference, let's try to add a note in `UPDATING.md` when doing these types of fixes that cause expected breaking changes.

**Pull request merged**